tests/std/test.lst

```diff
@@ -183,6 +183,7 @@ tests\GH_002058_debug_iterator_race
 tests\GH_002120_streambuf_seekpos_and_seekoff
 tests\LWG2597_complex_branch_cut
 tests\LWG3018_shared_ptr_function
+tests\LWG3527_uses_allocator_construction_args

```

Comment:
There already exists `P0475R1_P0591R4_uses_allocator_construction`. Tons of tiny tests aren't good for performance.